charmhelpers/contrib/storage/linux/utils.py

```diff
@@ -123,6 +123,7 @@ def mkfs_xfs(device, force=False, inode_size=1024):
     cmd = ['mkfs.xfs']
     if force:
         cmd.append("-f")
-
-    cmd += ['-i', "size={}".format(inode_size), device]
+    if inode_size:
```

Comment:
Can you amend the doc string above so it makes it clear inode_size can be set None to accept the xfs default setting ?

Reply:
@FrancescoDeSimone not sure if you saw my last comment. We should use `mkfs_xfs(device, force=False, inode_size=None)`